ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -5439,3 +5439,106 @@ AHU_BSPC_DX4SC_ECON_EFSS_EFVSC_SFSS_SFVSC_SSPC_RHHC_NONCANONICAL_1:
   - heating_water_valve_percentage_command
   - min_zone_air_relative_humidity_sensor
   - min_zone_air_relative_humidity_setpoint
+
+# TW-NTC-TPKE
+AHU_DSPM_DFVSC_DTM_RDC_OAMC_OA_CHWRC_RCO2M:
+  description: "AHU with discharge air/fan, chilled water valve operation, humidity monitoring with dual dampers monitoring and control ."
+  is_canonical: true
+  implements:
+  - AHU
+  - DSPM
+  - DFVSC
+  - DTM
+  - RDC
+  - OAMC
+  - OA
+  - CHWRC
+  - RCO2M
+  opt_uses:
+    - filter_alarm
+    - low_differential_pressure_alarm
+
+AHU_DFVSC_HWDC_UV_OAFM_MTM_CHWPVM_RDC_RCO2M_RTM:
+  description: "AHU with discharge air/fan, UV, chilled water valve and return air temperature monitoring with dual dampers monitoring."
+  is_canonical: true
+  implements:
+  - AHU
+  - DFVSC
+  - HWDC
+  - OAFM
+  - MTM
+  - CHWPVM
+  - RCO2M
+  - RTM
+  opt_uses:
+    - filter_alarm
+    - return_air_damper_percentage_sensor
+    - ultraviolet_lamp_run_status
+
+
+AHU_DFVSC_CHWPVM_RTM:
+  description: "AHU with discharge air/fan, chilled water valve and return air temperature monitoring."
+  is_canonical: true
+  implements:
```

Comment:
You have no temperature control type here

Reply:
I have added a temperature control type.

Reply:
you need to pair temperature control with the valve, if the valves control that way...